README.md

```diff
@@ -1076,7 +1077,7 @@ def example(db: Database): Int < IOs =
 // Another 'Cut' implementation
 val countTimesTen =
   new Cut[Database, Int, IOs] {
-    def apply[S](v: Database < S)(f: Database => Int < IOs) =
+    def apply[S](v: Database < S)(f: Database => Int < IOs)(using Trace) =
```

Comment:
Why does the readme need to mention `Trace` at all?

Reply:
I thought it'd be nice to ensure aspects propagate traces but, rethinking it now, it doesn't seem to make sense. We do want to show the trace based on the user's provided implementation for `apply`.